app/src/main/java/com/google/android/soundchecker/dsd/DsfReader.kt

```diff
@@ -145,26 +145,27 @@ class DsfReader(inputStream: InputStream) {
         }
     }
 
-    private fun readNextBlock(channel: Int): Boolean {
-        Arrays.fill(mData[channel], 0, mData[channel].size - 1, 0x0.toByte())
+    /**
+     * Returns the number of data read. Returns negative value if there is an error when reading or
+     * when it reaches the end of file and mark is not supported.
+     */
+    private fun readNextBlock(channel: Int): Int {
+        Arrays.fill(mData[channel], 0, mData[channel].size - 1, 0xA.toByte())
```

Comment:
0xA == 0x0A
I think we need 0xAA for a full byte of DSD silence.

Reply:
Or 0x55 would also work and not have any sign extension issues.

Reply:
Per discussed, we should use 0x69 as silence.